Modules/Sources/WordPressUI/Views/Users/Views/UserDetailsView.swift

```diff
@@ -62,15 +69,34 @@ struct UserDetailsView: View {
                 }
             }
         }
+        .alert(
+            Strings.setNewPasswordActionTitle,
+            isPresented: $presentPasswordAlert,
+            actions: {
+                SecureField(Strings.newPasswordPlaceholder, text: $newPassword)
+                SecureField(Strings.newPasswordConfirmationPlaceholder, text: $newPasswordConfirmation)
+                Button(Strings.updatePasswordButton) {
+                    Task {
+                        try await self.actionDispatcher.setNewPassword(id: user.id, newPassword: newPassword)
+                    }
+                }
+                .disabled(newPassword.isEmpty || newPassword != newPasswordConfirmation)
+                Button(role: .cancel) {
+                    presentPasswordAlert = false
+                } label: {
+                    // TODO: Replace with `SharedStrings.Button.cancel`
+                    Text(NSLocalizedString("shared.button.cancel", value: "Cancel", comment: "A shared button title used in different contexts"))
```

Comment:
would've probably been faster to do it that to add a TODO :laugh

Reply:
Ah, right. Yeah, I'd suggest moving it until there is an idea of how to organize it better.